src/content/test/sequence/columns.tsx

```diff
@@ -0,0 +1,111 @@
+        <p>
+            When white space characters are used to arrange text visually to create the appearance of columns, sighted users perceive a
+            structure that isn't represented in the code. Because assistive technologies present content in DOM order (the order that
+            content appears in the HTML), people who use assistive technologies are likely to read the text in an order that doesn't make
+            sense. (Also, a screen reader might announce each space character individually.)
+        </p>
+
+        <h2>How to fix</h2>
+        <p>
+            Use one of the following techniques to restructure the content so that the DOM order matches the expected reading order:
+            <ul>
+                <li> Good: Modify the content so it has no visually apparent columns. </li>
+                <li> Better: Use a layout table to display the content in columns.</li>
+                <li> Best: Use CSS to display the content in columns.</li>
+            </ul>
+        </p>
+
+        <Markup.PassFail
+            failText={
+                <p>
+                    White space characters in a <Markup.Code>{'<pre>'}</Markup.Code> element are used to create the appearance of two
+                    columns. When read in DOM order, the content doesn't make sense: "We the People general of the United Welfare, and
+                    States, in secure the Order…"{' '}
+                </p>
+            }
+            failExample={
+                <pre>
+                    <CodeExample>
+                        {`<pre>
+[We the People     promote the general
+of the United     Welfare, and
+States, in        secure the
+Order to form a   Blessings of
+more perfect      Liberty to
+Union,            ourselves and
+establish         our Posterity,
+Justice, insure   do ordain and
+domestic          establish this
+Tranquility,      Constitution
+provide for the   for the United
+common defense,   States of America.]
+</pre>`}
+                    </CodeExample>
+                </pre>
+            }
+            passText={<p>CSS is used to display the content in columns in a way that the DOM order matches the expected reading order.</p>}
+            passExample={`<style>
+            * {box-sizing: border-box;}
+            .column {float:left; width:50%;
+            padding:10px;}
+            </style>
+            ...
+            <div class="column">[We the People
+            of the United States,
+            in Order to form a
+            more perfect Union,
+            establish Justice,
+            insure domestic Tranquility,
+            provide for the common defense,
+            ]</div>
+            <div class="column">[promote the
+            general Welfare,
+            and secure
+            the blessings of Liberty
+            to ourselves and our Posterity,
+            do ordain and establish this Constitution
+            for the United States of America.]</div>
+            </body>
```

Comment:
Should we drop `</body>` and `</html>` here? We're not showing the opening markup for these elements.

Reply:
we have the ... on the top, not sure if its bad we don't have the opening markup. I am fine either way